### DISCUSSION TITLE
Disable notifications temporarily

### DISCUSSION THREAD

**Discussion by wetware05** [hluk/CopyQ#2213] [General]
Hi.

Wow, this has been changed. I don't know where I am, nor how to search.

Is there a command or shortcut to temporarily disable notifications? There are times when I use a script (in other programs) that clutters the clipboard and notifications of copied items become uncomfortable.

**Comment by wetware05** [comment]
Why does it show that there is a comment, but I don't see it? (???)

**Comment by dbareis** [comment]
The above is a bit cryptic but, I don't get notifications when clipboard is added to on Windows, I'd check for an CopyQ option that might cause that...

**Comment by wetware05** [comment]
Hi dbareis.

That option is the settings, notifications. Use Windows notifications. But on some systems Windows does not enable CopyQ and you have to enable it.

It's fine for me. So I know when the clipboard is used. But there are times I'd like to disable it (when I'm copying over and over). If the author created an "action" with a keyboard shortcut you could have that option

**Comment by hluk** [comment]
`copyq config clipboard_notification_lines 0` would effectively disable the automated clipboard notifications. The re-enable, replace `0` with a positive integer (as a maximum number of lines in the notification).

**Comment by wetware05** [comment]
Hi, hluk.

I don't know if you mean that this change must be done in the copyq.ini file or that an action can be done with "copyq config clipboard_notification_lines 0". I was referring to an action or a keyboard shortcut to change the state. If it is not possible, you could implement it in future versions.

**Reply by hluk** [reply]
The command can be added in Command dialog with a custom assigned shortcut:

```ini
[Command]
Command="
    copyq:
    const lines = 3;
    const currentValue = config('clipboard_notification_lines');
    const newValue = (lines == currentValue) ? 0 : lines;
    config('clipboard_notification_lines', newValue);"
GlobalShortcut=meta+n
Icon=\xf4b3
IsGlobalShortcut=true
Name=Toggle Clipboard Notifications
```

([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them))

Or you can go to Shortcuts tab in Preferences dialog and assign a shortcut to "Disable clipboard storing".

**Reply by wetware05** [reply]
It does not work. Well, for now I'll use the option to disable the clipboard. Thank you, hluk.
